parcllabs/services/parcllabs_service.py

```diff
@@ -199,17 +200,23 @@ def error_handling(self, response: requests.Response) -> None:
             error_details = response.json()
             error_message = error_details.get("detail", "No detail provided by API")
             error = error_message
+            
             if response.status_code == 403:
```

Comment:
could we create a function or method for handling the error codes and manage the different errors in there?

Reply:
kicking due to convo - method name is `error_handling` and currently accomplishes this.